src/pagination/Pagination.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { PureComponent, ReactElement } from 'react';
+import {
+    I18nManager,
+    Platform,
+    View,
+    StyleProp,
+    ViewStyle
+} from 'react-native';
+import PaginationDot from './PaginationDot';
+import styles from './Pagination.style';
+import type Carousel from 'src/carousel/Carousel';
+
+const IS_IOS = Platform.OS === 'ios';
+const IS_RTL = I18nManager.isRTL;
+
+type PaginationProps = {
+  activeDotIndex: number;
+  dotsLength: number;
+  activeOpacity?: number;
+  carouselRef?: Carousel<unknown> | null;
```

Comment:
` carouselRef?: Carousel<any> | null;` I think we can replace `unknown ` with `any`. Please share your thoughts regarding this.

Reply:
I tried avoiding all "any's". I suppose I could also template this so that it would be TData if you prefer

Reply:
@Titozzz Let's go with `TData` if you don't mind :-)